locksmith/src/utils/keyData.ts

```diff
@@ -17,13 +17,21 @@ export default class KeyData {
       if (!owner) {
         return {}
       }
+      const lock = await web3Service.getLock(lockAddress, network)
+      if (lock.publicLockVersion >= 10) {
+        key.expiration = await web3Service.getKeyExpirationByTokenId(
+          lockAddress,
+          tokenId,
+          network
+        )
+      } else {
+        key.expiration = await web3Service.getKeyExpirationByLockForOwner(
+          lockAddress,
+          owner,
+          network
+        )
+      }
       key.owner = owner
```

Comment:
Let's move this _above_ the expiration so that it's set even when the retrieval of expiration fails please.